workload/src/main/java/com/google/finapp/WorkloadMain.java

```diff
@@ -0,0 +1,154 @@
+// limitations under the License.
+
+package com.google.finapp;
+
+import com.google.protobuf.ByteString;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.StatusRuntimeException;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+
+/**
+ * A workload generator for the finance sample app that creates traffic by sending gRPC requests to
+ * the server.
+ */
+public final class WorkloadMain {
+  private static final String DEFAULT_ACCOUNT_BALANCE = "10000";
+  private static final String DEFAULT_TRANSFER_AMOUNT = "20";
+  private static final Logger logger = Logger.getLogger(WorkloadMain.class.getName());
+
+  private static class WorkloadGenerator {
+    private final ManagedChannel channel;
+    private final List<ByteString> ids = new ArrayList<>();
+
+    WorkloadGenerator(ManagedChannel channel) {
+      this.channel = channel;
+    }
+
+    void seedData(int numAccounts) {
+      int numFailedCreateAccounts = 0;
+      for (int i = 0; i < numAccounts; i++) {
+        try {
+          ByteString response =
+              WorkloadClient.getWorkloadClient(channel)
+                  .createAccount(
+                      DEFAULT_ACCOUNT_BALANCE,
+                      CreateAccountRequest.Type.CHECKING,
+                      CreateAccountRequest.Status.ACTIVE);
+          ids.add(response);
+        } catch (StatusRuntimeException e) {
+          numFailedCreateAccounts++;
+          logger.log(
+              Level.WARNING,
+              String.format("CreateAccount failed. Total count: %d", numFailedCreateAccounts));
+        }
+      }
+    }
+
+    void startSteadyLoad() {
+      Random random = new Random();
+      int numIds = ids.size();
+      if (numIds == 0) {
+        throw new IllegalStateException("No accounts were created successfully.");
+      }
+      while (true) {
+        ByteString fromId = ids.get(random.nextInt(numIds));
+        ByteString toId = ids.get(random.nextInt(numIds));
+        if (fromId.equals(toId)) {
+          continue;
+        }
+        WorkloadClient.getWorkloadClient(channel)
+            .moveAccountBalance(fromId, toId, DEFAULT_TRANSFER_AMOUNT);
+      }
+    }
+  }
+
+  public static void main(String[] args) {
+    CommandLine cmd = parseArgs(args);
+    String addressName = cmd.getOptionValue("a");
+    int port;
+    int numAccounts;
+    try {
+      port = ((Number) cmd.getParsedOptionValue("p")).intValue();
+      numAccounts = ((Number) cmd.getParsedOptionValue("n")).intValue();
```

Comment:
The signature of `getParsedOptionValue` returns an `Object`, but it is the type as defined for the `Option` (in this case, `Number`). 

See [these examples](https://www.tabnine.com/code/java/methods/org.apache.commons.cli.CommandLine/getParsedOptionValue) (especially the second-on) for common use cases.